ci-script/tsc-strict-error.mjs

```diff
@@ -0,0 +1,59 @@
+const dirToCheck = [
+    'compiler',
+    'runtime',
+]
+const tsErrorsThreshold = [
+    883,  /* 'compiler' */
+    358,  /* 'runtime' */
+];
+
+import { promisify } from 'util';
+import { exec as execCallback } from 'child_process';
+const exec = promisify(execCallback);
+
+const identity = i => i;
+
+async function main() {
+    const errors = await Promise.all(dirToCheck.map(dir => getTsErrors(dir)));
+    const errorsCount = errors.map(countTsErrors);
+    const noop = () => void 0;
+
+    let conditionallyPrintError = [noop, noop];
+    let conditionallyExplainError = noop;
+    let printErrorCount = [noop, noop];
+    let conditionallyAskForFix = noop;
+    let conditionallyExitWithNonZeroCode = noop;
+
+    for (let i = 0; i < tsErrorsThreshold.length; i++) {
+        if(errorsCount[i] > tsErrorsThreshold[i]) {
```

Comment:
```suggestion
        if (errorsCount[i] > tsErrorsThreshold[i]) {
```

Reply:
Happy to comply, thought I thought linter should catch this, not human...

Reply:
yeah, the linter only runs against `src` and `test` though. You'd have to add this directory and `.mjs` extension to the `lint` command in `package.json`. could be worth doing. maybe it would catch other stuff as well